router/middleware/header.go

```diff
@@ -33,6 +33,7 @@ func Options(c *gin.Context) {
 		c.Header("Access-Control-Allow-Headers", "authorization, origin, content-type, accept")
 		c.Header("Allow", "HEAD,GET,POST,PUT,PATCH,DELETE,OPTIONS")
 		c.Header("Content-Type", "application/json")
+		// nolint: gomnd // ignore magic number
```

Comment:
shouldn't we change to use `http.StatusOK` here?

Reply:
Will add another PR to address 👍

Reply:
@wass3r this should be complete with https://github.com/go-vela/worker/pull/150